cprover_bindings/src/env.rs

```diff
@@ -58,7 +65,7 @@ pub fn machine_model_symbols(mm: &MachineModel) -> Vec<Symbol> {
         ),
         int_constant("__CPROVER_architecture_wchar_t_width", mm.wchar_t_width),
         int_constant("__CPROVER_architecture_word_size", mm.word_size),
-        int_constant("__CPROVER_rounding_mode", mm.rounding_mode),
+        int_constant_c_int("__CPROVER_rounding_mode", mm.rounding_mode),
```

Comment:
Seriously? There is one remaining outlier? Is there an issue open to CBMC about that?

Reply:
Couldn't find one, so I opened https://github.com/diffblue/cbmc/issues/7282 (and pasted the issue there)

Reply:
I'll comment on the CBMC issue, but indeed rounding mode is different.